tasks/InfracostComment/task.json

```
@@ -9,9 +9,10 @@
   "author": "Infracost",
   "version": {
     "Major": 0,
-    "Minor": 1,
+    "Minor": 2,
     "Patch": 18
```

Should it be 0 as we bump the minor version?
```suggestion
    "Patch": 0
```

---

👍

---

done in https://github.com/infracost/infracost-azure-devops/pull/45/commits/51e7994d795d728acc2fcd1bef0007f8f0344df2